vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -147,6 +177,10 @@ public void remove() {
 
         private UpdateQueue(int size) {
             enqueue("$connector.updateSize", size);
+
+            // Update the server side property with the latest value of items
+            // count
+            getElement().setProperty("size", size);
```

Comment:
Uhm, why is this needed ? Please answer in code comment

Reply:
This is done to provide a use case, when you call `dataView.setItemCountEstimate(42)` and want to see the scroller size change immediately after open the dropdown. Even if the assumed size changed, the requested range is still [0, 50] and size changes are not committed to client. Grid has this line of code also in the `UpdateQueue`. Comment to a code is not probably reflect the meaning of this change

Reply:
Done